Grinnell-Events-iOS/Grinnell-Events-iOS/ViewController.m

```diff
@@ -347,7 +279,88 @@ - (void)searchBarCancelButtonClicked:(UISearchBar *)SearchBar {
 
 - (IBAction)didTapDays:(id)sender {
 }
+
 - (IBAction)goToToday:(id)sender {
     [self goToTodayAnimated:YES];
 }
+
+#pragma mark - Utility Methods
+- (void)showErrorAlert:(NSError *)error {
```

Comment:
Should we consolidate to one error message that recommends relaunching the app? We can work on better error handling in a future PR.